CHANGELOG.md

```diff
@@ -1,5 +1,21 @@
 # Changelog
 
+# [1.2.0] - 2023-03-06
+
+### Added
+- Common: Detection for at/hash/slash pattern in text
+- Common: API for replacing a detected pattern with link/text
+- Web & iOS: Support for inserting mentions
+- iOS: Support for inserting commands
```

Comment:
Hopefully we'll be able to reduce this to a single `Web & iOS` line before merging